### PR TITLE
Ekr review

### PR DESCRIPTION
**Pull request by ekr**
This includes

1. Removing my postal address. Mozilla closed that office.
2.  small editorial changes
3. Removing bogus whitespace.

### CODE REVIEW

**Comment by ekr** [comment]
@juberti PTAL

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -130,8 +123,8 @@
         exchanges.</t>
         <t>Regarding ICE
         <xref target="RFC8445" format="default"/>, JSEP decouples the ICE state
-        machine from the overall signaling state machine, as the ICE
-        state machine must remain in the JSEP implementation, because
+        machine from the overall signaling state machine. The ICE
+        state machine must remain in the JSEP implementationo because
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml:127]: Implementation

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -1217,8 +1210,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host ]]></sourcecode>
           contains an SDP answer per <xref target="RFC3264" format="default"/> with the supported
           configuration for the session that is compatible with the
           parameters supplied in the most recent call to
-          setRemoteDescription, which <bcp14>MUST</bcp14> have been called prior to
-          calling createAnswer. Like createOffer, the returned blob
+          setRemoteDescription (setRemoteDescription <bcp14>MUST</bcp14> have been called prior to
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml:1213]: What's the intent here?

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -2463,7 +2456,7 @@ as specified in <xref target="RFC5245" format="default"/>.</li>
           a=recvonly ]]></sourcecode>
           <t>The example in <xref target="sec.detailed-example" format="default"/> shows a more involved case of "LS" group
           generation.</t>
-          <t>The next step is to generate "m=" sections for each "m="
+          <t>The next step is to generate one "m=" section for each "m="
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml:2459]: Maybe one->a

**Review by juberti — CHANGES_REQUESTED** [review]

File: draft-ietf-rtcweb-jsep.xml (hunk 2 above)
Comment (ekr) [reply draft-ietf-rtcweb-jsep.xml:1213]: I just felt like it was run-on. Don't feel strongly about this.

Comment (juberti) [reply draft-ietf-rtcweb-jsep.xml:1213]: I see what you're getting at, but I'm not crazy about the parenthetical. I'm inclined to leave it as-is here.

Comment (ekr) [reply draft-ietf-rtcweb-jsep.xml:1213]: I am also not crazy about the parenthetical. If we can't improve it, I'm OK with it as-is.

**Review by fluffy — APPROVED** [review]
LGTM

File: draft-ietf-rtcweb-jsep.xml (hunk 2 above)
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml:1213]:
```suggestion
          setRemoteDescription; setRemoteDescription <bcp14>MUST</bcp14> have been called prior to
```

File: draft-ietf-rtcweb-jsep.xml [hunk 4]
```diff
@@ -1217,8 +1210,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host ]]></sourcecode>
           contains an SDP answer per <xref target="RFC3264" format="default"/> with the supported
           configuration for the session that is compatible with the
           parameters supplied in the most recent call to
-          setRemoteDescription, which <bcp14>MUST</bcp14> have been called prior to
-          calling createAnswer. Like createOffer, the returned blob
+          setRemoteDescription (setRemoteDescription <bcp14>MUST</bcp14> have been called prior to
+          calling createAnswer.) Like createOffer, the returned blob
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml:1214]:
```suggestion
          calling createAnswer. Like createOffer, the returned blob
```

File: draft-ietf-rtcweb-jsep.xml (hunk 2 above)
Comment (juberti) [reply draft-ietf-rtcweb-jsep.xml:1213]: PTAL at my suggestion of a semicolon

Comment (ekr) [reply draft-ietf-rtcweb-jsep.xml]: LGTM.

**Comment by ekr** [comment]
@juberti I think I've got your changes and we are ready to land?

**Review by juberti — APPROVED** [review]

**Pull request merged**